client/_index.html

```diff
@@ -7,9 +7,9 @@
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/0.5.0/modern-normalize.min.css" integrity="sha256-N6+kUxTWxpqVK+BrPWt3t4jeOWPtp37RZEbm5n9X+8U=" crossorigin="anonymous" />
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css" integrity="sha256-46qynGAkLSFpVbEBog43gvNhfrOj+BmwXdxFgVK/Kvc=" crossorigin="anonymous" />
 
-  <!-- <script src="https://cdn.jsdelivr.net/npm/jshint@2.11.0/dist/jshint.min.js"></script> -->
-  <script src="https://cdn.jsdelivr.net/npm/jsonlint@1.6.3/web/jsonlint.min.js"></script>
-  <script src="https://cdn.jsdelivr.net/npm/csslint@1.0.5/dist/csslint.min.js"></script>
+  <!-- <script src="{{jsdelivrCdnUrl}}/npm/jshint@2.11.0/dist/jshint.min.js"></script> -->
+  <script src="{{jsdelivrCdnUrl}}/npm/jsonlint@1.6.3/web/jsonlint.min.js"></script>
+  <script src="{{jsdelivrCdnUrl}}/npm/csslint@1.0.5/dist/csslint.min.js"></script>
```

Comment:
I am not sure is this necessary. Are you trying to run this app without internet connection?

Reply:
This was mainly to be consistent with the idea of being able to "swap" jsdelivr.